save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -434,8 +434,8 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
 
     private fun RBuilder.renderSettings() {
         child(organizationSettingsMenu) {
-            attrs.organization = state.organization!!
-            attrs.selfRole = Role.SUPER_ADMIN
+            attrs.organizationName = props.organizationName
+            attrs.selfRole = Role.VIEWER
```

Comment:
I can see that this prop is not used anywhere, is it needed?

Reply:
what is the `attrs.selfRole`

Reply:
It used to be useful but now we do not need It. Removed it from both project and organization.